tests/cli_test.py

```diff
@@ -245,7 +245,7 @@ def test_calendar(runner):
         assert not result.exception
         assert result.exit_code == 0
         output = '\n'.join([
-            "    Mo Tu We Th Fr Sa Su     No events",
```

Comment:
Not sure I'm convinced about rendering removing this one -- we're rendering the calendar anyway, so it looks kinda odd that there's nothing on the side.

I can't imagine this output being parsed by scripts either; it's far from ideal for being parsed.